.appveyor.yml

```diff
@@ -8,9 +8,9 @@ cache:
 
 branches:
   only:
-    - master
+    - 3.x
     - 3.next
-    - 4.x
```

Comment:
should not we keep the `master` ?

Reply:
i dont think so for 3.x ones.

Reply:
Appveyor might have only one configuration for the project.